contrib/drivers/lvm/metrics_cli.go

```diff
@@ -0,0 +1,143 @@
+// Copyright (c) 2019 The OpenSDS Authors.
+//
+//    Licensed under the Apache License, Version 2.0 (the "License"); you may
+//    not use this file except in compliance with the License. You may obtain
+//    a copy of the License at
+//
+//         http://www.apache.org/licenses/LICENSE-2.0
+//
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+//    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+//    License for the specific language governing permissions and limitations
+//    under the License.
+package lvm
+
+import (
+	log "github.com/golang/glog"
+	"github.com/opensds/opensds/pkg/utils/exec"
+	"regexp"
+	"strings"
+)
+
+const (
+	sarNotEnabledOut = "Please check if data collecting is enabled"
+	cmdNotFound      = "No such file or directory"
+)
+
+type MetricCli struct {
+	// Command executer
+	BaseExecuter exec.Executer
+	// Command Root executer
+	RootExecuter exec.Executer
+}
+
+func NewMetricCli() (*MetricCli, error) {
+	return &MetricCli{
+		BaseExecuter: exec.NewBaseExecuter(),
+		RootExecuter: exec.NewRootExecuter(),
+	}, nil
+}
+
+func (c *MetricCli) execute(cmd ...string) (string, error) {
+	return c.RootExecuter.Run(cmd[0], cmd[1:]...)
+}
+
+func is_sar_enabled(out string) bool {
```

Comment:
s/is_sar_enabled/isSarEnabled

Reply:
What do you mean by using `out` here?

Reply:
> s/is_sar_enabled/isSarEnabled

Changed to camel case

Reply:
> What do you mean by using `out` here?

out is used here to check whether the output of command contains ""Please check if data collecting is enabled" , this means sar is not enabled